src/pages/index.js

```diff
@@ -137,13 +187,31 @@ function addCardToList(element, method = "prepend") {
   cardList[method](card);
 }
 
-// Render initial cards
-function renderCards() {
-  initialCards.forEach((element) => {
-    addCardToList(element, "append");
+function renderUserProfile({name, about, avatar}) {
+  profileName.textContent = name;
+  profileVocation.textContent = about;
+  avatarElement.src = avatar;
+}
+
+function renderCards(cards) {
+  cards.forEach((card) => {
+    addCardToList(card, "append");
   });
 }
 
+// Render initial cards
+function callApi() {
```

Comment:
`callApi`


You get the initial app data here


`getAppData` or `getInitialData`